metrixpp/ext/std/tools/view.py

```diff
@@ -544,7 +544,7 @@ def cout_txt_regions(path, regions, indent = 0):
                 if field == '__diff__':
                     continue
                 if field in list(diff_data.keys()):
-                    diff_str = " [" + ("+" if diff_data[field] >= 0 else "") + str(diff_data[field]) + "]"
+                    diff_str = " [" + ("+" if diff_data[field] >= 0 else "") + "{0:.3g}".format(diff_data[field]) + "]"

```

Comment:
I had to change this in order to get the same output using python2 or python3. 

For example, the problem using str was that it was printing 0.07500000000000001 using python2 and 0.075 using python3.

[ I am not sure it is the right to solve this pb, and/or if it should be used elsewhere too. ]